### ISSUE TITLE
Menu should be scrollable

### ISSUE THREAD

**Issue by bfirsh** [boxwise/boxwise#206]
The menu doesn't scroll and we're right on the borderline of what can fit in the menu on an iPhone 5 sized screen. With things like #23, it's going to get too long.

The menu should be scrollable, somehow.

The menu is a Material UI drawer. Here are the docs: https://material-ui.com/demos/drawers/

**Comment by jclarkin** [comment]
@bfirsh I tested this in Chrome resized for mobile, and the menu drawer does allow scrolling. In which browser did you encounter this issue?

**Comment by bfirsh** [comment]
Ah - interesting. Mobile Safari. This is me recalling it not working from a few weeks ago, so maybe it is fixed now. Let me double check.

**Comment by bfirsh** [comment]
Yeah - it doesn't seem to work on mobile safari. When you swipe up and down on the menu, it scrolls the page behind it, not the menu. Makes it rather hard to get to the "sign out" button...

**Comment by acournoyer1** [comment]
I'll grab this one, literally just fixed 2 or 3 instances of it on our site

**Comment by bfirsh** [comment]
This has been fixed on iOS but still up for grabs on iOS - see #220

**Comment by spielhoelle** [comment]
Cannot reproduce. Menu scrolls on chrome, ff, iphone mobile safari. On Iphone it seemed to be a bit buggy but worked in the end.
